init.d/modules.BSD.in

```diff
@@ -0,0 +1,24 @@
+#!@SBINDIR@/openrc-run
+
+description="Loads a user defined list of kernel modules."
+
+depend() {
+	after bootmisc
+	keyword -jail -prefix
+}
+
+start()
+{
+	local cnt=0 x
+	for x in $modules
+	do
+		ebegin "Loading module $x"
+		kldload "$x";
+		eend $? "Failed to load $x" && : $(( cnt += 1 ))
```

Comment:
I realize the Linux modules initscript does this, but it's really rather gross.  I think we can get just rid of the count entirely.